src/Atmos/Model/boundaryconditions.jl

```diff
@@ -88,14 +88,14 @@ struct InitStateBC <: BoundaryCondition
 end
 function atmos_boundary_state!(::Rusanov, bc::InitStateBC, m::AtmosModel,
                                stateP::Vars, auxP::Vars, nM, stateM::Vars,
-                               auxM::Vars, bctype, t, _...)
-  init_state!(m, stateP, auxP, auxP.coord, t)
+                               auxM::Vars, bctype, t, args...)
+  init_state!(m, stateP, auxP, auxP.coord, t, args...)
```

Comment:
I don't think you want this: 
1) some boundary conditions get the interior points as extra args
2) the extra args aren't going to be passed in anyway